_targets.R

```diff
@@ -161,19 +161,21 @@ list(
                    barheight = 2.4,
                    text_size = 25)
   ),
-  
+
   # Flow timeseries nationally - Instagram 
   tar_target(
     flow_national_instagram_png,
-    national_ig(check_ig_safezone = TRUE,
-                file_png = "out/flow_national_ig.png",
-                file_ig_grid = "out/flow_national_ig_grid.png",
+    national_ig(file_png = "out/flow_national_ig.png",
```

Comment:
SC: Now that so much is happening here, and so little is happening in `cowplot_national_explainer()` I don't know that you really need to have this split into two separate targets. Is there a use case any longer for the version returned by `national_ig()`?

Reply:
Good question - I guess there's not but I suppose if IG changes formats down the line we can do all the small adjustments in that target and then final touches in `cowplot_final_national_ig()`, so I'll leave it for now.